aws/cloud-init/master.script.tpl

```diff
@@ -294,16 +279,16 @@ EOF
                 "Action": "UPSERT",
                 "ResourceRecordSet": {
                     "AliasTarget": {
-                        "DNSName": "${cluster_name}",
+                        "DNSName": "${ops_advertise_addr}",
```

Comment:
What if `ops_advertise_addr` is not set? It also means these won't be done for regular clusters which is I'm not sure what we want. I would keep it backwards-compatible, you can for example set the cluster name to the DNS name e.g. `--cluster=example.gravitational.io`.

Also, `ops_advertise_addr` can contain port number so many of the places where it's used will actually be invalid I think.

Reply:
I will introduce a new variable.

I changed it from `cluster_name` as the cluster name and ops address are different in our case.